lib/blocktree/blocktree.go

```diff
@@ -330,6 +330,19 @@ func (bt *BlockTree) GetAllBlocks() []Hash {
 	return bt.root.getAllDescendants(nil)
 }
 
+// GetAllDescendants returns all blocks that are descendants of the given block.
+func (bt *BlockTree) GetAllDescendants(a common.Hash) ([]Hash, error) {
+	bt.RLock()
+	defer bt.RUnlock()
+
+	an := bt.getNode(a)
```

Comment:
Please rename `a` -> `blockHash` and `an` to `node` (or something else, I'm not sure what `an` should be here.